FAQ.md

```diff
@@ -38,37 +38,34 @@ languages.
 
-## What is the commercial licensing model?
+## What are the commercial licensing options?
 
-The basic principle behind our commercial licensing is that you start for free and pay when you're shipping.
+We offer 3 options under the commercial license - Ambassador, Flex and Buyout. All 3 options include a non-exclusive, perpetual, irrevocable, non-transferable right to use Slint.
 
-We offer a perpetual license option and we generally deploy a per-product license, regardless of how many developers, designers, Q&A engineers are using Slint.
+The Ambassador license is a free license that can be provided in lieu of helping us promote Slint.
 
-If this doesn't fit you, don't hesitate to contact us and we'd be happy to work together to find a solution.
+The Flex license is a per User-Seat license (with updates included as long as the subscription is active) to develop apps with Slint.
+Distribution of such apps requires additional fees.
 
-You can find a more detailed overview of our commercial licensing and the pricing at <https://slint-ui.com/pricing.html>.
+The Buyout license is a volume-based buyout license that includes unlimited User-Seats (with updates included) to develop apps with Slint as well as distribute such apps up to the purchased volume.
 
 ## Can I try Slint using the GPL version and then switch to the commercial license later?
 
-Yes. The GPL is a distribution license that applies only when you ship your application. You can
-evaluate Slint and develop your product internally using the GPL license, and only acquire a commercial
-license when you want to ship your product. If you choose a per seat licensing model, the time spent
-developing needs to be accounted for. However, support for bug fixes requires a commercial license.
+Yes. You can evaluate Slint using the GPL license, and acquire the commercial license after the evaluation, with the option to transfer the development work from GPL to commercial for free.
 
 ## Is there a discount or reduction for independent developers or small businesses?
 
 Yes, check out our [Ambassador program](https://slint-ui.com/ambassador-program.html)
 
 ## I want to develop a free software product using Slint, but I don't like the GPL and prefer to use a non-copyleft license.
 
-You can still publish your own source code under a permissive license compatible with the GPL, such as BSD, MIT, or Apache license.
-The distribution of a binary or a package containing Slint still needs to be licensed under the GPL.
-It is up to those who want to distribute a non-free version of the application to acquire a commercial license.
+A couple of options could be:
+
+  * publish your own source code under a permissive license compatible with the GPL, such as BSD, MIT, or Apache license. However, the binary or the package
+    containing Slint needs to be licensed under GPL,
+  * consider of one of our [commercial licensing options](#what-are-the-commercial-licensing-options).
```

Comment:
I think this doesn't answer the question. nor is useful for the people for which i had the question in mind.
We're talking of the kind of people that think the GPL is too restrictive for an open source license. Of course the commertial licensing are not an option for them.  
The previous wording was better IMHO.

Reply:
I'm inclined to agree. The question implies the preference to use a non-copyleft license. Technically the commercial license is a license of that type, but it's unlikely that's meant here.

Reply:
removed the statement